docs/src/declarative_database_management.md

```diff
@@ -23,25 +21,81 @@ spec:
     name: cluster-example
 ```
 
-Once the reconciliation cycle is completed successfully, the `Database` 
-status will show a `ready` field set to `true` and an empty `error` field.
+Note that the Database references a Cluster, on which the database will be
+created.
+The Database object is managed by the instance manager of the cluster's
+primary instance. Declarative database management is not supported in replica
```

Comment:
Ah, interesting. OK on one hand, the Database object reconciliation would fail, and would be re-queued, so if eventually we promoted the cluster, the database reconciliation would proceed.
Now, in a different direction: this feature is not so much doing "management" as doing declarative creation. Once a manifest is successfully applied, it's over and done until we alter it (and the `generation` is incremented).
So, to understand your use case better: if we create Database Foo in the primary, then it will be replicated in the replica cluster, no? So having a copy of the Database manifest in the replica cluster would not be of any use.

Reply:
I've rewritten this, I hope it's now clearer.